pkg/controllers/networking/pod_controller_test.go

```diff
@@ -855,3 +1024,20 @@ func statefulOwnerReferenceRender() metav1.OwnerReference {
 		BlockOwnerDeletion: &blockOwnerDeletion,
 	}
 }
+
+func specifiedMACAddressPoolAnnotationRender(macPool []string) string {
```

Comment:
Is this function similar to `strings.Join`?

Reply:
Yes, I've replaced it with `strings.Join`.